helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
 class DrydockConfig(object):
```

Comment:
Does a config.py really serve us best? A config.ini would be easier for a helm chart for instance, to overlay into a `drydock` container.  Having it in code makes it a bit harder to manage.

Lots of https://docs.openstack.org/developer/oslo.config/configopts.html examples exist.